### PR TITLE
Adding code tabs library and attempting to build the staging site.

### PR DESCRIPTION
**Pull request by Serdaro**

### CODE REVIEW

**Review by emreyigit — DISMISSED** [review]

**Comment by ihsandemir** [comment]
@Serdaro Can we move this to Draft status since it is not final?

**Comment by zpqrtbnk** [comment]
Hey. For some reason this PR is doing weird things to our CI build. I *really* want to review it :) Is there any chance you could push a dummy commit or something, to force it to rebuild?

**Comment by Serdaro** [comment]
Just did @zpqrtbnk  :)

**Comment by ihsandemir** [comment]
verify

File: doc/modules/ROOT/pages/quickstart.adoc [hunk 1]
```diff
@@ -9,50 +9,70 @@ such as distributed maps, queues, topics and more. All, with enterprise level se
...
+
+|&#46;NET Framework and Core
+|Hazelcast .NET client is distributed as a NuGet package which targets .NET Standard versions 2.0 and 2.1.
 It can therefore be used in any application targeting .NET versions that support these .NET Standard versions:
-** https://dotnet.microsoft.com/en-us/download/dotnet-framework[.NET Framework] 4.6.2 and newer, on Windows
-** .NET Core https://dotnet.microsoft.com/en-us/download/dotnet/2.1[2.1] and https://dotnet.microsoft.com/en-us/download/dotnet/3.1[3.1], on Windows, Linux and MacOS
-* https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.2[PowerShell] shell and scripting language.
 
-The upcoming .NET 5 version supports .NET Standard 2.1, and therefore should execute the Hazelcast .NET client without issues, but that is not supported yet.
+* https://dotnet.microsoft.com/en-us/download/dotnet-framework[.NET Framework] 4.6.2 and newer, on Windows
```
Comment (emreyigit) [inline doc/modules/ROOT/pages/quickstart.adoc:33]:
```suggestion
* https://dotnet.microsoft.com/en-us/download/dotnet-framework[.NET Framework] 4.6.2 or 4.8, on Windows
```
Although we target netstandart2.0, we don't target 4.7. I've never tried with 4.7. So, it's better say 4.6.2 or 4.8.

File: doc/modules/ROOT/pages/quickstart.adoc [hunk 2]
```diff
@@ -9,50 +9,70 @@ such as distributed maps, queues, topics and more. All, with enterprise level se
...
+* .NET Core https://dotnet.microsoft.com/en-us/download/dotnet/2.1[2.1] and https://dotnet.microsoft.com/en-us/download/dotnet/3.1[3.1], on Windows, Linux and MacOS
```
Comment (emreyigit) [inline doc/modules/ROOT/pages/quickstart.adoc:34]:
```suggestion
* https://dotnet.microsoft.com/en-us/download/dotnet[7.0], on Windows, Linux and MacOS
```
Just a quick update. We support .Net 6 and above now.

File: doc/modules/ROOT/pages/quickstart.adoc [hunk 3]
```diff
@@ -9,50 +9,70 @@ such as distributed maps, queues, topics and more. All, with enterprise level se
...
+
+|Docker
+|https://docs.docker.com/get-docker/[Get Docker]
+|===
+
+WARNING: The upcoming .NET 5 version supports .NET Standard 2.1, and therefore should execute the Hazelcast .NET client without issues, but that is not supported yet.
```
Comment (emreyigit) [inline doc/modules/ROOT/pages/quickstart.adoc:40]: I'm not sure we should have this. If we will have that sentence ,we can say .Net 8.

**Pull request closed without merging**